staging/src/k8s.io/client-go/tools/clientcmd/config.go

```diff
@@ -398,6 +410,11 @@ func writeCurrentContext(configAccess ConfigAccess, newCurrentContext string) er
 		}
 		config.CurrentContext = newCurrentContext
 
+		if lockErrors[destinationFile] != nil {
```

Comment:
move this above the getConfigFromFile call

Reply:
Fixed in ecf3b83e88ef4c4791889f922514cf459d3b3058